### ISSUE TITLE
Update Gulp / Package.json file

### ISSUE THREAD

**Issue by jmslbam** [level-level/Clarkson-Theme#20]
- [x] Update outdated modules
- [x] Remove image optimalisation from flow - Update: we are keeping this.
- [x] Add SVG only optimalisation - Update: we are keeping this.

**Comment by jmslbam** [comment]
On Linux fsevents is skipped because it's an Darwin only package that fixes `watch` on Darwin.
Do we want to replace this with Chokidar to get a cross platform same behaviour?
https://www.npmjs.com/package/chokidar

@NielsdeBlaauw Maybe that could also be the reason the `watch` in the new branch doesn't work?

**Comment by jmslbam** [comment]
Nevermind, we don't include ourselves. It's an dependency of an other package 💃

**Comment by NielsdeBlaauw** [comment]
We have found that part of the problem with the `watch` part only exists when the process is running inside a vagrant box. We have verified that twig-change refreshes work when running the new branch outside of vagrant, but fails to update when run inside a virtual machine. We have not yet checked if the problem with adding new less files works the same.

The problem with the twig files is probably because the vagrant folder is a shared-folder from the host machine. Maybe the events are not triggering correctly on shared-folders.
